File: service/src/main/java/hu/bsstudio/scores/endresult/EndResultService.java
package hu.bsstudio.scores.endresult;

import hu.bsstudio.scores.endresult.model.EndResultedTeam;
import hu.bsstudio.team.model.DetailedTeam;
import reactor.core.publisher.Mono;

public interface EndResultService {
    Mono<DetailedTeam> updateEndResult(final EndResultedTeam endResultedTeam);
}

File: server/service/src/main/kotlin/hu/bsstudio/robonaut/scores/endresult/EndResultService.kt
package hu.bsstudio.robonaut.scores.endresult

import hu.bsstudio.robonaut.scores.endresult.model.EndResultedTeam
import hu.bsstudio.robonaut.team.model.DetailedTeam
import reactor.core.publisher.Mono

interface EndResultService {
  fun updateEndResultSenior(endResultedTeam: EndResultedTeam): Mono<DetailedTeam>

  fun updateEndResultJunior(endResultedTeam: EndResultedTeam): Mono<DetailedTeam>
}
